src/site/filters/liquid.js

```diff
@@ -19,23 +19,17 @@ module.exports = function registerFilters() {
   liquid.filters.humanizeTimestamp = dt =>
     moment.unix(dt).format('MMMM D, YYYY');
 
+  function prettyTimeFormatted(dt, format) {
+    const date = moment(dt).format(format);
+    return date.replace(/AM/g, 'a.m.').replace(/PM/g, 'p.m.');
+  }
+
   liquid.filters.timeZone = (dt, tz, format) => {
     if (dt && tz) {
-      const tzOffset = new Date(dt).getTimezoneOffset(tz) * 60000;
-      const dtDate = new Date(
-        dt.toLocaleString('en-US', {
-          timeZone: tz,
-        }),
-      ).getTime();
-
-      const diffToMoment = dtDate - tzOffset;
-
-      const prettyTime = moment(diffToMoment).format(format);
-      const prettyTimeFormatted = prettyTime
-        .replace(/AM/g, 'a.m.')
-        .replace(/PM/g, 'p.m.');
-
-      return prettyTimeFormatted;
+      const timeZoneDate = new Date(dt).toLocaleString('en-US', {
+        timeZone: tz,
+      });
+      return prettyTimeFormatted(timeZoneDate, format);
```

Comment:
Should this be a new inline function since it is only used once?

Reply:
Trivial, btw.

Reply:
do you mean `prettyTimeFormatted`?

Reply:
If so, it is used down below. That's why i abstracted it out. Was using the same methods to achive the same time formatted string in at least two places